src/inc/zapper.h

```diff
@@ -126,7 +126,11 @@ class Zapper
 
 #if defined(FEATURE_CORECLR) && !defined(FEATURE_MERGE_JIT_AND_ENGINE)
     SString                 m_CLRJITPath;
+    bool                    m_fDontLoadJit;
```

Comment:
When JITPath is not specified, CrossGen defaults to load JIT from the same location as CrossGen itself.